packages/locales/src/locales/en.json

```diff
@@ -208,5 +208,8 @@
   "unquoted-string": "an unquoted string",
   "unsorted-keys": "Unsorted keys",
   "uuid": "a UUID",
-  "vector": "a vector"
+  "vector": "a vector",
+  "parser.macro.key": "a macro key",
+  "parser.macro.at-least-one": "at least one macro argument",
+  "parser.macro.illegal": "Illegal key character: “%0%”"
```

Comment:
This should be scoped under `mcfunction.` and the file should be ordered by key lexicographically.